gen3/bin/kube-setup-secrets.sh

```diff
@@ -226,6 +226,27 @@ if [[ -f "${WORKSPACE}/${vpc_name}/creds.json" ]]; then # update peregrine secre
   fi
 fi
 
+if [[ -f "${WORKSPACE}/${vpc_name}_output/creds.json" ]]; then # update pidgin secrets
+  if [ ! -d "${WORKSPACE}/${vpc_name}" ]; then
```

Comment:
This is out of date - should be `"${WORKSPACE}/${vpc_name}/creds.json` - look at the other blocks